README.md

```diff
@@ -0,0 +1,108 @@
+# gmusic-ui.js
+[![Build Status](https://travis-ci.org/gmusic-utils/gmusic-ui.js.svg?branch=master)](https://travis-ci.org/gmusic-utils/gmusic-ui.js)
+[![GitHub release](https://img.shields.io/github/tag/gmusic-utils/gmusic-ui.js.svg)]()
+[![Code Climate](https://img.shields.io/codeclimate/github/gmusic-utils/gmusic-ui.js.svg)]()
+[![GitHub license](https://img.shields.io/github/license/gmusic-utils/gmusic-ui.js.svg)]()
+
+Browser-side JS library for controlling [Google Music][] through UI manipulation.
+
+[Google Music]: https://play.google.com/music/
+
+`gmusic-ui.js` is not created by, affiliated with, or supported by Google Inc.
+
+[Google Music]: https://play.google.com/music/listen
+
+## Getting Started
+It's important to note the `gmusic-ui.js` depends on [`gmusic.js`](https://github.com/gmusic-utils/gmusic.js) in order to operate correctly
+
+### npm
+Install the module with: `npm install gmusic.js gmusic-ui.js`
+
+Once installed, add it to your HTML and access it via `window.GMusic`.
+
+```html
+<script src="node_modules/gmusic.js/dist/gmusic.min.js"></script>
+<script src="node_modules/gmusic-ui.js/dist/gmusic-ui.min.js"></script>
```

Comment:
Hmmm, I totally copy-pasted this from somewhere else :+1: :laughing: 

I'll change it to

```html
<script>
require('gmusic.js');
require('gmusic-ui.js');
</script>
```